src/components/ReviewConfirm/Certify.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <v-card flat id="step-5-container">
-    <div class="certify-container">
+    <div class="certify-container" v-if="certifyStatementResource">
```

Comment:
Without this, there was a template error if ReviewConfirm was the initial page, because this prop isn't initially defined, so you can't de-reference it on lines 24, 30, and 38 below.